### ISSUE TITLE
broken new post with hugo 0.18

### ISSUE THREAD

**Issue by vhquang** [halogenica/beautifulhugo#20]
This can be reproduced with completely new site:
```bash
hugo new site testing
```
Post would be created successfully:
```bash
hugo new post/a.md
cat content/post/a.md
    +++
    date = "2017-03-01T16:36:09-08:00"
    title = "a"
    draft = true

    +++
```

But after install and setting the theme to `beautifulhugo`, new post become broken:
```bash
cd themes && git clone https://github.com/halogenica/beautifulhugo.git beautifulhugo && cd ..
echo 'theme = "beautifulhugo"' >> config.toml
hugo new post/b.md
cat content/post/b.md
    +++
    title = ""
    bigimg = ""
    date = "2014-07-11T10:54:24+02:00"
    subtitle = ""

    +++
```

Notice the `date` is wrong, and also the `title` is empty, and missing the `draft` flag.

This works fine with hugo 0.16, but starts breaking on > 0.18. My temporary solution is to remove the `theme` variable in the configuration file, and set it on the command line for every hugo command:
```bash
hugo server -t beautifulhugo -wD
```

If someone can give me a pointer on what could go wrong, I can try to get to the bottom of this, and maybe submit an PR.

**Comment by halogenica** [comment]
This behavior seems to have changed with the newest versions of hugo. The correct fix I believe is to update the archetypes. I've made a change to set the default archetype to define all the extra front matter features this theme takes advantage of, and no longer specify "title" or "date" which were conflicting with the hugo default front matter fields. Verified this is fixed with ba07f7cc67a0e3de0b20f36590cd70b3eec124fe
